include/mbedtls/ssl.h

```diff
@@ -1252,6 +1252,10 @@ struct mbedtls_ssl_session {
 
 #endif /*  MBEDTLS_SSL_PROTO_TLS1_3 && MBEDTLS_SSL_SESSION_TICKETS */
 
+#if defined(MBEDTLS_SSL_EARLY_DATA)
+    uint32_t MBEDTLS_PRIVATE(max_early_data_size);          /*!< maximum amount of early data in tickets */
```

Comment:
Early Data is a new feature that introduced by TLS 1.3. It may be used in future protocols(TLS 1.4 ...) . For the time being, we only allow early data in resumption session. But it is allowed in a PSK connection in RFC 8446.  So we does not guards it with MBEDTLS_SSL_PROTO_TLS1_3 and MBEDTLS_SSL_SESSION_TICKETS. 
For the implementation reason, we add check_config .